Bouncing particle example (#1351)
chaos-polymtl/lethe: doc/source/examples/dem/bouncing-particle/bouncing-particle.rst

Description of the Case
-------------------------

> This simulation consists of a single particle bouncing on a flat plane. The particle is inserted at rest at a specific position and accelerate due to the action of gravity. Upon reaching the outer limit of the domain, the particle-wall contact stops the particles form leaving the triangulation. Depending on the values of spring constant and restitution coefficient used, the particle loses kinetic energy, thus the height of the next bounce decreases until the particle comes to a complete stop.

Reviewer: ```suggestion
This simulation consists of a single particle bouncing on a flat plane. The particle is inserted at rest at a specific position and accelerates due to the action of gravity. Upon reaching the outer limit of the domain, the particle-wall contact stops the particles from leaving the triangulation. Depending on the values of spring constant and restitution coefficient used, the particle loses kinetic energy, thus the height of the next bounce decreases until the particle comes to a complete stop.
```

Insertion Info
~~~~~~~~~~~~~~~~~~

> Since the insertion of the particle must be done at as specific height, the ``list`` insertion method is used. The ``insertion frequency`` can be set to any value, since we're only using one particle. The insertion location is centered in the X-Y plane and at a height of 0.5 m in the Z direction.

Reviewer: ```suggestion
Since the insertion of the particle must be done at a specific location, the ``list`` insertion method is used. The ``insertion frequency`` can be set to any value, since we're only using one particle. The insertion location is centered in the X-Y plane and at a height of 0.5 m in the Z direction.
```

.. code-block:: text

    subsection insertion info
      set insertion method     = list
      set insertion frequency  = 10000
      set list x               = 1.0
      set list y               = 1.0
      set list z               = 0.5
    end

Lagrangian Physical Properties
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Different spring constant and restitution coefficient are used in this example. The spring constant is not defined explicitly in the parameter file, consequently the physical properties must be chosen to obtain a desired spring constant. The interested reader can consult `DEM theory <../../../theory/multiphase/cfd_dem/dem.html>`_ for more information about the definition of the sprint constant.

> The ``poisson ratio`` of both the wall and the particle are being set to 0.3 arbitrarily. Young's modulus of the wall is also being set arbitrarily to the value of \SI{1e12}{\pascal}.

Reviewer: ```suggestion
The ``poisson ratio`` of both the wall and the particle are being set to 0.3 arbitrarily. The Young's modulus of the wall is also being set arbitrarily to the value of \SI{1e12}{\pascal}.
```
